src/ai/ai_instance.cpp.in

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
Perhaps instead of autogenerating the entire file, it could include a "ai_includes.hpp" file that's built by cmake?